app/src/main/res/layout/activity_q_r_scan.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".view.activity.QRScanActivity">
+
+    <com.budiyev.android.codescanner.CodeScannerView
+        android:id="@+id/scanner_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:autoFocusButtonColor="@android:color/white"
+        app:autoFocusButtonVisible="true"
+        app:flashButtonColor="@android:color/white"
+        app:flashButtonVisible="true"
+        app:frameColor="@android:color/white"
+        app:frameCornersSize="50dp"
```

Comment:
Instead of using hardcoded values, use create/use resources. Same goes for other values too

Reply:
Done, replaced hardcoded values with apt resources

Reply:
> Done, replaced hardcoded values with apt resources

Please squash the commits into one single commit and always follow snake casing while naming the resources. Have a look at this article - [link](https://medium.com/mindorks/android-resource-naming-convention-42e4e8026614)